edit/linter-manager.js

```diff
@@ -328,9 +328,13 @@ linterMan.DEFAULTS = {
   }
 
   function createTable(cm) {
-    const caption = $create('caption');
-    const tbody = $create('tbody');
-    const table = $create('table', [caption, tbody]);
+    const caption = $create('td.caption');
+    const tbody = $create('tbody.report');
+    const table = $create('table',
+      $create('tr', [
+        caption,
+        $create('td', tbody),
```

Comment:
It seems that we don't need the outer table. We can use div for layout:
```html
<div style="display: flex; flex-direction: row">
  <div class="section-title"></div>
  <table> ... </table>
</div>
```

Reply:
Indeed, but switching to a div necessitates adding a CSS rule whereas using a table doesn't.

Reply:
Then let's switch to div and CSS

Reply:
But why?

Reply:
So we don't have to use table

Reply:
Not sure what do you mean:

* If you ask why table without `<table>` is wrong, here is the spec: https://html.spec.whatwg.org/multipage/tables.html#the-tbody-element
    > As a child of a [table](https://html.spec.whatwg.org/multipage/tables.html#the-table-element) element, after any [caption](https://html.spec.whatwg.org/multipage/tables.html#the-caption-element), [colgroup](https://html.spec.whatwg.org/multipage/tables.html#the-colgroup-element), and [thead](https://html.spec.whatwg.org/multipage/tables.html#the-thead-element) elements, but only if there are no [tr](https://html.spec.whatwg.org/multipage/tables.html#the-tr-element) elements that are children of the [table](https://html.spec.whatwg.org/multipage/tables.html#the-table-element) element.
    
* If you ask why using table for layout is wrong, technically, it isn't. It is still the main layout technique for email. However, we have other options (float/flex/grid/etc) for general websites nowadays. You may find some articles discussing table layout drawbacks with search engine.

  For me, it is cleaner to use div + CSS. People won't ask "why do we have to use table here" when seeing the code. And for consistency since we use CSS for other stuff.

Reply:
I found a way to simplify the structure and code even more so now it's a div.